vscode-ext/src/tests/e2e.ts

```diff
@@ -0,0 +1,146 @@
+    const fileName = "app_flask.py";
+    const mirrordConfigPath = join(testWorkspace, '.mirrord/mirrord.json');
+    const configurationFile = "Python: Current File";
+    const breakPoint = 9;
+    const DEFAULT_CONDITION_TIMEOUT = 10000;
+
+    before(async function () {
+        console.log("podToSelect: " + podToSelect);
+        console.log("kubeService: " + kubeService);
+
+        expect(podToSelect).to.not.be.undefined;
+        expect(kubeService).to.not.be.undefined;
+
+        browser = VSBrowser.instance;
+        // need to bring the flask app in open editors
+        await browser.openResources(testWorkspace, join(testWorkspace, fileName));
+    });
+
+    after(async function () {
+        if (await debugToolbar?.isDisplayed()) {
+            await debugToolbar.stop();
+        }
+    });
+
+    it("enable mirrord", async function () {
+        const statusBar = new StatusBar();
+        await browser.driver.wait(async () => {
+            const enableButton = await statusBar.getItem("Enable mirrord");
+            if (enableButton !== undefined) {
+                enableButton.click();
+                return true;
+            }
+        }, DEFAULT_CONDITION_TIMEOUT, "mirrord `enable` button not found -- timed out");
+
+        await browser.driver.wait(async () => {
+            const enableButton = await statusBar.getItem("Disable mirrord");
+            if (enableButton !== undefined) {
+                return true;
+            }
+        }, DEFAULT_CONDITION_TIMEOUT, "Mirrord not enabled -- timed out");
+    });
+
+    it("create mirrord config", async function () {
+        // gear -> $(gear) clicked to open mirrord config
+        const statusBar = new StatusBar();
+        await browser.driver.wait(async () => {
+            const mirrordSettingsButton = await statusBar.getItem("gear");
+            if (mirrordSettingsButton !== undefined) {
+                mirrordSettingsButton.click();
+                return true;
+            }
+        }, DEFAULT_CONDITION_TIMEOUT, "mirrord config `$(gear)` button not found -- timed out");
+
+        await browser.driver.wait(async () => {
+            return await existsSync(mirrordConfigPath);
+        }, DEFAULT_CONDITION_TIMEOUT, "mirrord `default` config not found");
+    });
+
+    it("select pod from quickpick", async function () {
+        await setBreakPoint(fileName, breakPoint, browser);
+        await startDebugging(configurationFile);
+
+        const inputBox = await InputBox.create();
+        // assertion that podToSelect is not undefined is done in "before" block   
+        await browser.driver.wait(async () => {
+            return await inputBox.isDisplayed();
+        }, DEFAULT_CONDITION_TIMEOUT, "quickPick not found -- timed out");
+        await inputBox.selectQuickPick(podToSelect!);
+    });
+
+    it("wait for breakpoint to be hit", async function () {
+        debugToolbar = await DebugToolbar.create(20000);
```

Comment:
Is this the timeout? Why is it different from the default?

Reply:
needs more than 10 seconds